modules/swagger-parser/src/main/java/io/swagger/parser/util/SwaggerDeserializer.java

```diff
@@ -756,6 +756,12 @@ public Model definition(ObjectNode node, String location, ParseResult result) {
                 impl.setUniqueItems(bp);
             }
 
+            BigDecimal bd = getBigDecimal("minimum", node, false, location, result);
```

Comment:
including PR #444 in order to fix the relative path used in #444

After merging the PR #444, I can update this PR to fix the merge conflict (result of the previous merge)